homeassistant/components/devolo_home_network/strings.json

```
@@ -21,11 +23,18 @@
       },
       "zeroconf_confirm": {
         "description": "Do you want to add the devolo home network device with the hostname `{host_name}` to Home Assistant?",
-        "title": "Discovered devolo home network device"
+        "title": "Discovered devolo home network device",
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "password": "Password you protected the device with."
```

This can use a reference
```suggestion
          "password": "[%key:component::devolo_home_network::config::user::data_description::password%]"
```